### DISCUSSION TITLE
auto-crop

### DISCUSSION THREAD

**Discussion by sasharogers31** [sumatrapdfreader/sumatrapdf#3529] [Ideas]
Hey, I really wanted a feature to automatically crop comics according to content. For example, crop white margins. Thank you very much for this app!

**Comment by GitHubRulesOK** [comment]
Pixel Autocrop is complex and usually documents (including comics) are uncropped as the margins are a native multicoloured feature, (why else are they included?) the simplest way to Autocrop scanned speckles as pixels is use a graphics editor like IrfanView which can autocrop them in bulk, by ignoring minor tonal variations in "white" spaces.

**Comment by GitHubRulesOK** [comment]
NOTE if the pages have white space margins the new trimmed view mode may work with those but the effects can be very variable so the upper 2 are trimmed and the next 2 are not trimmed 
<img width="1434" height="839" alt="image" src="https://github.com/user-attachments/assets/ca7288fd-3878-4082-9200-bb1db88164b1" />

**Comment by kjk** [comment]
If you have examples where it doesn't work right, please open a bug and provide a test file (or send me via e-mail).

This is a heuristic that works by looking at pixels so it might require tweaking, but I can only fix issues I can reproduce.
